nixos/maintainers/scripts/ec2/amazon-base-config.nix

```diff
@@ -1,5 +1,5 @@
 { modulesPath, ...}:
 {
-  imports = [ "${modulesPath}/virtualisation/amazon-config.nix" ];
+  imports = [ ../../../../nixos/modules/virtualisation/amazon-image.nix ];
```

Comment:
In my context, `<nixpkgs>` was wrong because it's getting evaluated to build the image (and I was running it in a git clone) in my specific test (see me calling `eval-config` in the main test script). Perhaps that's just the wrong approach?